packages/app/src/app/overmind/namespaces/pro/actions.ts

```diff
@@ -3,6 +3,14 @@ import { Context } from 'app/overmind';
 import { withLoadApp } from 'app/overmind/factories';
 import { Step, Plan, PaymentSummary, PaymentPreview } from './types';
 
+export const getPrices = async ({ state, effects }: Context) => {
+  try {
+    state.pro.prices = await effects.api.prices();
+  } catch (err) {
+    // Fail silently.
```

Comment:
is this a common scenario? should we track it in sentry?

Reply:
@alexnm I don't think so but I also didn't want to trigger notifications since the price is just a footer label and is not a blocker of the experience. Should the back-end have the logs if these requests fail or is the front-end's responsibility to tell that the request has failed?

Reply:
makes sense, if it's not on the critical path, no logging is needed